src/TimeGrid.js

```diff
@@ -189,22 +195,56 @@ export default class TimeGrid extends Component {
             ref={gutterRef}
             className='rbc-time-gutter'
           />
-
-          {this.renderEvents(range, rangeEvents, this.props.now)}
+          {eventsRendered}
 
         </div>
       </div>
     );
   }
+  renderEventsWithResource(range, events, today, resources) {
```

Comment:
can this method be joined with `renderEvents` I'd be worried the common code will diverge unintentionally